src/site/content/en/blog/webtransport/index.md

```diff
@@ -20,104 +20,89 @@ feedback:
   - api
 ---
 
-{% Aside 'caution' %} This proposal has undergone significant changes since the
-start of the Origin Trial. Starting with Chrome 87, WebTransport has replaced
-QuicTransport as the top-level interface that developers interact with.
-
-As a result, some of the information, and all of the sample code, in this
-article is out of date. For the latest on this evolving proposal, please read
-refer to the
-[editor's draft of WebTransport](https://w3c.github.io/webtransport/). It
-includes an [Examples](https://w3c.github.io/webtransport/#examples) section
-with updated code snippets.
-
-Once the proposal stabilizes, we will update this article and associated code
-samples with up to date information.
-{% endAside %}
-
 ## Background
 
-### What's QuicTransport?
+### What's WebTransport?
 
-[QuicTransport](https://wicg.github.io/web-transport/#quic-transport) is a web API that uses the [QUIC](https://www.chromium.org/quic) protocol in a bidirectional, non-HTTP transport. It's intended for two-way communications between a web client and a QUIC server. It supports sending data both unreliably via its [datagram APIs](#datagram), and reliably via its [streams APIs](#stream).
+[WebTransport](https://w3c.github.io/webtransport/) is a web API that uses the [HTTP/3](https://quicwg.org/base-drafts/draft-ietf-quic-http.html) protocol as a bidirectional transport. It's intended for two-way communications between a web client and an HTTP/3 server. It supports sending data both unreliably via its [datagram APIs](#datagram), and reliably via its [streams APIs](#stream).
```

Comment:
Should we have a little comment that says something like "formerly known as QuicTransport", or do you think it's obvious?

Reply:
There's an `Aside` that goes into detail about that at https://github.com/GoogleChrome/web.dev/pull/5029/files#diff-b8c695635c07b4dffc40aa857ab06919d35d0217dfa6dba4c817c0c1411daa20R43-R47

I think that should suffice, rather than also calling it out here?

Reply:
The aside seems fine to me. I'd take Pete's side if this were a name change for something in production like change from 'Feature Policy' to 'Permissions Policy'.